solidity-v1/dashboard/src/css/withdraw-granted-tokens.less

```diff
@@ -0,0 +1,101 @@
+.withdraw-granted-tokens {
+  &__form_checkbox {
+  }
+
+  &__total-ready-to-release-amount {
+    display: inline;
+  }
+
+  &__grants-accordion {
+    width: 100%;
+    border: 1px solid @color-grey-30;
+    border-radius: 0.5rem;
+    margin: 0.5rem 0;
+
+    .accordion {
+      &__button {
+        border-bottom: none;
+        align-items: center;
+
+        .withdraw-granted-tokens__expand-button {
+          display: inline-block;
+          content: "";
+          height: 7px;
+          width: 7px;
+          margin-top: 5px;
+          border-bottom: 1px solid black;
+          border-right: 1px solid black;
+          transform: rotate(-135deg);
+        }
+      }
+
+      &__button[aria-expanded="true"] > .withdraw-granted-tokens__expand-button,
+      &__button[aria-selected="true"]
+        > .withdraw-granted-tokens__expand-button {
+        transform: rotate(45deg);
+        transform-origin: 100% 40%;
+        margin-top: 3px;
+      }
+
+      //&__button:after {
+      //  display: inline-block;
+      //  content: "";
+      //  height: 5px;
+      //  width: 5px;
+      //  margin-top: 5px;
+      //  border-bottom: 2px solid black;
+      //  border-right: 2px solid black;
+      //  transform: rotate(-135deg);
+      //}
+      //
+      //&__button[aria-expanded="true"]::after,
+      //&__button[aria-selected="true"]::after {
+      //  transform: rotate(45deg);
+      //  transform-origin: 100% 40%;
+      //}
+
+      &__panel {
+        padding: 0.1rem 1rem 0.5rem 1rem;
+      }
+    }
+  }
+
+  &__details-text {
+    margin-left: auto;
+    margin-right: 0.5rem;
+    font-size: 14px;
+    font-weight: 400;
+  }
+
+  &__info-row {
```

Comment:
I believe we should add `align-items: center`.

Reply:
Fixed in abc3c313d11754156ad366c85075a44a99709f7c